pof/src/types.rs

```diff
@@ -218,6 +218,17 @@ impl Display for Vec3d {
         write!(f, "{}, {}, {}", &self.x, &self.y, &self.z)
     }
 }
+/// The orientation of the model before rotation
+#[derive(Clone, Copy)]
+pub enum ModelOrientation {
+    Up,
+    Forward,
+}
+impl Default for ModelOrientation {
+    fn default() -> Self {
+        ModelOrientation::Up
+    }
+}
```

Comment:
What is this enum for? It doesn't seem like `to_rotation_matrix()` uses anything other than 'Up'?